libpod/oci.go

```diff
@@ -367,8 +367,6 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 	args := []string{}
 
 	// TODO - should we maintain separate logpaths for exec sessions?
```

Comment:
Is this comment still valid

Reply:
It will no longer be once i finish conmon exec :)

Reply:
It will no longer be once i finish conmon exec :)